include/slang.h

```diff
@@ -254,15 +254,16 @@ convention for interface methods.
 
 // GCC Specific
 #if SLANG_GCC_FAMILY
-
 #	define SLANG_NO_INLINE __attribute__((noinline))
 #	define SLANG_FORCE_INLINE inline __attribute__((always_inline))
 #   define SLANG_BREAKPOINT(id) __builtin_trap();
 #	define SLANG_ALIGN_OF(T)	__alignof__(T)
+#endif // SLANG_GCC_FAMILY
 
+#if SLANG_GCC_FAMILY || defined(__llvm__)
```

Comment:
`__builtin_offsetof` should also work with MSVC, should we be adding it as-well to avoid compiler hacks?
godbolt link: https://godbolt.org/z/cGfWfz3nP

```suggestion
#if SLANG_GCC_FAMILY || defined(__llvm__) || defined(_MSC_VER)
```

Reply:
I am reluctant to adopt the suggested change because there isn't any problem with the current implementation.
I am going to skip the suggested change for now and we can add it when it becomes a problem.
Personally I am more comfortable using the `((T*)0)->elem` workaround, because that's what I have been doing on almost all projects in the past, if not all.